src/main/java/de/caritas/cob/messageservice/api/helper/HttpUrlUtils.java

```diff
@@ -0,0 +1,14 @@
+package de.caritas.cob.messageservice.api.helper;
+
+import static org.apache.commons.lang3.Validate.notNull;
+
+
+public class HttpUrlUtils {
```

Comment:
unit test missing. If possible, maybe you can find library with this functionality 
instead.

Reply:
done